junifer/api/tests/test_cli.py

```diff
@@ -74,6 +83,59 @@ def test_run_and_collect_commands(
     assert collect_result.exit_code == 0
 
 
+@pytest.mark.parametrize(
```

Comment:
This is not fully testing all the options.

What if the user wants to specify `subject` and `session`? Can you test that, it should be a comma-separated list.

Reply:
Should be addressed now.